homeassistant/components/switch/__init__.py

```diff
@@ -134,7 +134,6 @@ def toggle(hass, entity_id=None):
 
 class SwitchDevice(ToggleEntity):
     """Representation of a switch."""
-
```

Comment:
Please revert this change.

Reply:
Done, not sure why it's not flagging this.

Reply:
Doesn't seem to be done. 😉